pkg/rtc/participant.go

```diff
@@ -399,7 +399,7 @@ func (p *ParticipantImpl) Start() {
 	})
 }
 
-func (p *ParticipantImpl) Close() error {
+func (p *ParticipantImpl) Close(bool) error {
```

Comment:
argument is not named and seems not used? Is it for future use?

Reply:
oops, good catch. fixed it locally but forgot to push up.